CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 - Expand template variable in Array of Any [#651](https://github.com/yonaskolb/XcodeGen/pull/651) @kateinoigakukun
 - Significantly improve performance when running with a large number files. [#658](https://github.com/yonaskolb/XcodeGen/pull/658) @kateinoigakukun
+- Less `Array()`. [#661](https://github.com/yonaskolb/XcodeGen/pull/661) @RomanPodymov
```

Comment:
We can maybe move this into an “Internal” section and say `Removed needless Array initialization`

Reply:
Hello @yonaskolb 
Done.